index.js

```diff
@@ -11,6 +11,14 @@ var onExit = require('signal-exit')
 var stripBom = require('strip-bom')
 var SourceMapCache = require('./lib/source-map-cache')
 var resolveFrom = require('resolve-from')
+var crypto = require('crypto')
+
+function md5 (str) {
+  return crypto
+    .createHash('md5')
+    .update(str, 'utf8')
+    .digest('hex')
```

Comment:
I couldn't find `rev-hash` when I wrote this, we could switch to that. I wonder, Is there any advantage to truncating to 10 characters in a temp folder?

// @sindresorhus